### PR TITLE
Add `Address`, `Notification`, `Tax`, and `User` service to the deployment

### PR DESCRIPTION
**Pull request by delvh**
Additionally, prepare everything for the remaining deployments so that in the subsequent PRs, only a single file must be changed.

Fixes https://frontend.gropius.duckdns.org/components/2571e2de-a0e7-4017-b721-2fd3692ad5d6/issues/8df0f88a-e1b5-450d-9cf3-043f650a7793 (address)
Fixes https://frontend.gropius.duckdns.org/components/2571e2de-a0e7-4017-b721-2fd3692ad5d6/issues/81ce9e7d-ebfb-4bad-bb55-47062e4ea1f1 (notification)
Fixes https://frontend.gropius.duckdns.org/components/2571e2de-a0e7-4017-b721-2fd3692ad5d6/issues/a75f8681-3ce3-48bc-9279-e8c5e5638290 (tax)
Fixes https://frontend.gropius.duckdns.org/components/2571e2de-a0e7-4017-b721-2fd3692ad5d6/issues/8011380a-5157-4bc4-9b73-2b93372c965e (user)

## DoD

- [x] Requirements of the issue are met
- [x] `terraform apply` works
- [x] `kubectl get pod --namespace misarch` afterwards shows all pods up and running

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ It is not intended for productive use in the slightest.
 
 - **Disappearing Dapr Sidecars**: If Dapr sidecars disappear, causing communication to stop working in the cluster, try restarting the affected deployments.
 - **Schema Changes in Services**: If there are schema changes in individual services without changes in the gateway code, a restart of the gateway deployment is required.
-- **\<x\> exists already**: When running `terraform apply` and this error occurs, execute `kubectl delete namespaces misarch` (or whatever you named your namespace) and try again. We know it's weird, but we did not find a better solution for it.
+- **\<x\> exists already**: When canceling a previous `terraform apply` and re-running `terraform apply`, this error can occur. It means that `terraform` sees state outside of its control. In this case, you have two options: `terraform refresh` may help sometimes. If it does not help, delete the component and try again. In the worst case, execute `kubectl delete namespaces misarch` (or whatever you named your namespace) and try again. We know it's weird, but it seems to be caused by Terraforms design.
 -
```
Comment (nk-coding) [inline README.md]: is there any content for this point?

File: adding-a-new-service.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# How to add a new service
```
Comment (nk-coding) [inline adding-a-new-service.md]: imho this should be parts of our docs, not sure if it is needed here additionally

Comment (nk-coding) [reply adding-a-new-service.md]: maybe put it in the docs and put a link in the README?

File: dbs.tf [hunk 3]
```diff
@@ -19,3 +41,309 @@ resource "helm_release" "misarch_catalog_db" {
...
+      username: "${var.MISARCH_DB_USER}"
+      database: "${var.MISARCH_DB_DATABASE}"
+      password: "${random_password.misarch_shipment_db_password.result}"
+    metrics:
+      enabled: true
+    EOF
+  ]
+}
+
+resource "helm_release" "misarch_shoppingcart_db" {
```
Comment (nk-coding) [inline dbs.tf]: ohm, shoppingcart (and all other rust services) use mongo

**Review by nk-coding — DISMISSED** [review]

File: dbs.tf (hunk 3 above)
Comment (delvh) [reply dbs.tf]: Done

File: configmaps.tf [hunk 4]
```diff
@@ -24,3 +53,200 @@ resource "kubernetes_config_map" "misarch_catalog_env_vars" {
...
+}
+
+resource "kubernetes_config_map" "misarch_shoppingcart_env_vars" {
+  metadata {
+    name      = local.misarch_shoppingcart_env_vars_configmap
+    namespace = local.namespace
+  }
+
+  data = {
+    "SPRING_R2DBC_URL"      = "r2dbc:postgresql://${local.shoppingcart_db_url}/${var.MISARCH_DB_DATABASE}"
```
Comment (nk-coding) [inline configmaps.tf]: these variables make no sense for the rust services

Comment (delvh) [reply configmaps.tf]: Done

**Review by nk-coding — APPROVED** [review]

**Pull request merged**